src/com/itacademy/eshop/product/MobilePayment.java

```diff
@@ -0,0 +1,20 @@
+package com.itacademy.eshop.product;
+
+public class MobilePayment extends Payment {
+    private String phoneNumber;
```

Comment:
private field is not set and no setter, you should either include it in a constructor or create a setter for it